.github/workflows/docs_test.yml

```diff
@@ -76,8 +76,9 @@ jobs:
     - name: Install node dependencies
       run: |
         cd docs/test
-        npm cache clean --force
-        npm ci
+        npm cache clean -f
+        rm package-lock.json
```

Comment:
do we still have this file if the package-lock.json is deleted from git

Reply:
Yeah we do... deleted!